src/ripple/app/paths/PathRequests.cpp

```diff
@@ -249,7 +249,25 @@ PathRequests::makeLegacyPathRequest(
         app_.getLedgerMaster().newPathRequest();
     }
 
-    return result.second;
+    return std::move (result.second);
+}
+
+Json::Value
+PathRequests::doLegacyPathRequest (
+        Resource::Consumer& consumer,
+        std::shared_ptr<ReadView const> const& inLedger,
+        Json::Value const& request)
+{
+    auto cache = std::make_shared<RippleLineCache> (inLedger);
+
+    auto dummyCompletion = []() { ; };
+    auto req = std::make_shared<PathRequest> (app_, std::move (dummyCompletion),
+        consumer, ++mLastIdentifier, *this, mJournal);
+
+    auto result = req->doCreate (cache, request);
+    if (result.first)
+        result.second = req->doUpdate (cache, false);
```

Comment:
The dummy completion ensures we get old-style pathfinding, which doesn't have fast updates.
```
        if (! hasCompletion() && valid)
            doUpdate(cache, true);
```
Since `hasCompletion()` will be true, there will be no call to `doUpdate`.